packages/gatsby/src/commands/build-html.ts

```diff
@@ -200,12 +206,55 @@ const renderHTMLQueue = async (
 
   try {
     await Bluebird.map(segments, async pageSegment => {
-      const renderHTMLResult = await renderHTML({
+      const renderHTMLResult = (await renderHTML({
         envVars,
         htmlComponentRendererPath,
         paths: pageSegment,
         sessionId,
-      })
+      })) as IRenderHtmlResult
```

Comment:
I think casting it here can be a bit misleading because in the case of `renderHTMLDev` the `renderHTMLResult` will be something different.

Can we cast it inside `if (isPreview) {}` block instead, same as we do on line 260 below?

Reply:
ah good call 👍